pkg/cmd/target/unset_test.go

```diff
@@ -211,6 +187,35 @@ var _ = Describe("Command", func() {
 		Expect(currentTarget.SeedName()).To(BeEmpty())
 		Expect(currentTarget.ShootName()).To(BeEmpty())
 	})
+
+	It("should be able to unset targeted control plane", func() {
+		streams, _, out, _ := util.NewTestIOStreams()
+
+		// user has already targeted a garden, project, shoot and control plane
+		currentTarget := target.NewTarget(gardenName, projectName, "", shootName).WithControlPlane(true)
+
+		fakeGardenClient := fake.NewClientBuilder().WithObjects(project, shoot).Build()
+
+		// setup command
+		targetProvider := internalfake.NewFakeTargetProvider(currentTarget)
+		clientProvider := internalfake.NewFakeClientProvider()
+		clientProvider.WithClient(gardenKubeconfig, fakeGardenClient)
+
+		factory := internalfake.NewFakeFactory(cfg, nil, clientProvider, nil, targetProvider)
+		cmd := cmdtarget.NewCmdUnset(factory, cmdtarget.NewUnsetOptions(streams))
```

Comment:
This code is repeated in every single test. It should be moved into a `JustBeforeEach` hook. Maybe not as part of this PR but in general.

Reply:
Yes maybe you saw that I already cleaned up this file, also all the test data has been redeclared in every single test. I moved this in the before each function, however skipped the other lines (for now) as I did not want to change the implementation itself. Also, this pattern is repeated in other files e.g. target_test.go. Let's do this in a separate PR.

Reply:
🆗